frontend/src/components/Worksheet/ScheduleSuggestionsPreview.module.css

```
@@ -0,0 +1,111 @@
+.addedRow {
+  align-items: center;
+  display: flex;
+  gap: 0.75rem;
+  justify-content: space-between;
+  margin-top: 0.75rem;
+}
+
+.addedSummary {
+  color: var(--color-text-secondary);
+  min-width: 0;
+  word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)